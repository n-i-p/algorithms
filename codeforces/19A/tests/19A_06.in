8
T00
T01
T02
T03
T04
T05
T06
T07
T01-T00 97:82
T02-T00 48:26
T02-T01 16:18
T03-T00 71:10
T03-T01 98:7
T03-T02 82:58
T04-T00 67:0
T04-T01 49:21
T04-T02 82:59
T04-T03 33:8
T05-T00 71:70
T05-T01 94:59
T05-T02 3:59
T05-T03 52:53
T05-T04 67:31
T06-T00 65:39
T06-T01 27:10
T06-T02 13:20
T06-T03 42:93
T06-T04 24:48
T06-T05 78:92
T07-T00 41:31
T07-T01 65:71
T07-T02 77:41
T07-T03 55:0
T07-T04 68:49
T07-T05 96:32
T07-T06 3:91
