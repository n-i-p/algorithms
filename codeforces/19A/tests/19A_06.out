T03
T04
T05
T07
